### DISCUSSION TITLE
Logo is not loading properly in 5.0 version

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#637] [General Discussion]
While publishing the content the comment section logos are not loading properly in the 5.0 version 
In the 4.6 version, it is loading properly
I have attached screenshots from both versions for reference
If there is any fix for this, please suggest us. Thanks
**This screenshot is from the 4.6 version**

![image](https://github.com/Sunbird-Ed/Community/assets/109504832/58d4f48f-9962-4600-a78d-b9d3531942ef)

**This screenshot is from 5.0 version**

![image](https://github.com/Sunbird-Ed/Community/assets/109504832/1e80e2c1-9720-4699-b74e-c5606c2a491d)

@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

**Comment by vinukumar-vs** [comment]
@sachinb80 Did you verified the logo's path is right in the network tab. Is it failing in the network tab or not.

@rajeevsathish Meanwhile can you check & share the details..

**Reply by sachinb80** [reply]
@vinukumar-vs It is not failing in the network tab and the Path is the same 
**This screenshot is from 4.6 version where logo is visible**
![image (16)](https://github.com/Sunbird-Ed/Community/assets/109504832/193c3916-9671-4041-bfd2-fd68b05b633c)

**And this screenshot is from 5.0 version**

![image (17)](https://github.com/Sunbird-Ed/Community/assets/109504832/a7579339-a4b7-43bc-8a8c-149552dd5dd4)

**Comment by Krishnaj20** [comment]
@rajeevsathish - are you looking into this?

**Comment by ashokreddy1208** [comment]
@sachinb80 We created a bug in jira for this issue - https://project-sunbird.atlassian.net/browse/CO-639

**Reply by sachinb80** [reply]
@ashokreddy1208 Ok. Please update us after any progress. Thanks

**Comment by ashokreddy1208** [comment]
@sachinb80 will update you once issue is resolved.

**Reply by sachinb80** [reply]
Ok

**Comment by ashokreddy1208** [comment]
@sachinb80 We created a bug in Jira (https://project-sunbird.atlassian.net/browse/CO-639) hence closing this thread
